### PR TITLE
Create ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57710](https://cumulocity.atlassian.net/browse/MTM-57710)
Original PR: [6882](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6882)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: global roles application and microservice view (#6855) [GRAFT][release/cd] (#6882)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Global roles can now view applications and microservices
```

File: content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57710
+version: 1020.16.1
+---
+global roles application and microservice view (#6855) [GRAFT][release/cd] (#6882)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, users with global roles were unable to view applications and microservices in the Cumulocity IoT platform. This change now allows users with global roles to view applications and microservices, providing them with a more comprehensive overview of the platform's components. This improvement impacts all users with global roles, enabling them to access and monitor applications and microservices without requiring additional permissions.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md (hunk 1 above)
Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md:3]:
```suggestion
title: Application access section of each Global role will only show Applications and not Microservices
```

File: content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md (hunk 2 above)
Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md:17]:
```suggestion
Previously under Application access of each global role, Applications and Microservices used to show up, but now the improvement has been made to show only Applications  under "Application access" and not Microservices
```

Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md:17]:
```suggestion
Previously, the **Application access** list of a global role showed both applications and microservices. This has been changed to only show applications in the *Application access** list.
```

**Review by BeateRixen — APPROVED** [review]

**Review by kgol-c8y — APPROVED** [review]

**Pull request merged**